### DISCUSSION TITLE
Issue with running fds

### DISCUSSION THREAD

**Discussion by bjmali3389** [firemodels/fds#12697] [Q&A] [answered]
Hi
I am a graduate student currently trying to learn about fds and pyrosim. I ran the couch test file with a multiple layer of polyurethane and it worked fine.  Now i am trying to model the couch with two different materials for upholstery which when heated by a square burner at 15kw for 180 s, burns . I looked up the example of polyurethane and wood example in the guide and tried to input the values as such. However, since I was trying to model it being burned due to an external flux from the burner, I used a layered surface instead of supply and I tried to provide the burner as heater surface. I am unable to run the file due to errors. Could you kindly guide what mistakes i might have done and how to correct it. Thank you in advance

I am getting the following errors
Starting FDS ...

 MPI Process      0 started

 Reading FDS input file ...

forrtl: severe (59): list-directed I/O syntax error, unit -5, file Internal List-Directed Read
Image              PC                Routine            Line        Source             
fds_openmp.exe     00007FF702155867  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF70211A9AB  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF709641208  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF709647D77  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF7095C16E7  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF70924F431  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF70920FEEE  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF709B85AA0  Unknown               Unknown  Unknown
KERNEL32.DLL       00007FFA7CB8257D  Unknown               Unknown  Unknown
ntdll.dll          00007FFA7D9AAA58  Unknown               Unknown  Unknown
[runfds.exe] subprocess exit: 59

**FDS input file**

couchex3.fds
Generated by PyroSim 2023.3.1312
Mar 28, 2024, 5:43:58 PM

&HEAD CHID='couchex3', TITLE='Single Couch Test Case'/
&TIME T_END=600.0/
&DUMP DT_DEVC=5.0, DT_HRR=5.0, DT_RESTART=1000.0, MASS_FILE=.TRUE./
&MISC POROUS_FLOOR=.FALSE./
&COMB SUPPRESSION=.FALSE./

&MESH ID='Mesh01[0,0,0]', IJK=50,50,24, XB=0.0,5.0,0.0,5.0,0.0,2.4/

&SPEC ID='SOOT', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='NITROGEN', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='CARBON DIOXIDE'/
&SPEC ID='WATER VAPOR', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='OXYGEN', LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='CARBON MONOXIDE'/
&SPEC ID='Polyester', FORMULA='CC1H1.7O0.74N0.002.', SPECIFIC_HEAT=1.3/
&SPEC ID='POLYURETHANE', FORMULA='C25H42O6N2', SPECIFIC_HEAT=1.76/
&SPEC ID='Air', BACKGROUND=.TRUE., 
  SPEC_ID(1)='NITROGEN',
  SPEC_ID(2)='OXYGEN', 
  VOLUME_FRACTION(1)=3.76,
  VOLUME_FRACTION(2)=1.0/
&SPEC ID='products_wood', 
  SPEC_ID(1)='CARBON DIOXIDE',
  SPEC_ID(2)='CARBON MONOXIDE',
  SPEC_ID(3)='NITROGEN',
  SPEC_ID(4)='SOOT',
  SPEC_ID(5)='WATER VAPOR', 
  VOLUME_FRACTION(1)=0.96,
  VOLUME_FRACTION(2)=3.6E-3,
  VOLUME_FRACTION(3)=3.838,
  VOLUME_FRACTION(4)=0.0351,
  VOLUME_FRACTION(5)=0.848/

&PART ID='ignitor particle',
      STATIC=.TRUE.,
      SURF_ID='ignitor',
      QUANTITIES='PARTICLE TOTAL HEAT FLUX',
      AGE=180.0/

&REAC ID='Reaction1',
      FYI='Polyurathane reaction',
      HEAT_OF_COMBUSTION=2270.0,
      FUEL='POLYURETHANE',
      SPEC_ID_NU='Air','POLYURETHANE','products_wood',
      NU=-27.23,-1.0,1.0/
&REAC ID='Reaction2',
      FYI='Reac_polyester',
      HEAT_OF_COMBUSTION=2580.0,
      FUEL='Polyester',
      SPEC_ID_NU='Air','CARBON DIOXIDE','CARBON MONOXIDE','Polyester','products_wood',
      NU=-1.02,-1.0,-1.0,-1.0,1.0/

&MATL ID='GYPSUM PLASTER',
      SPECIFIC_HEAT=1.0,
      CONDUCTIVITY=0.5,
      DENSITY=500.0/
&MATL ID='FABRIC',
      FYI='Polyster: approx',
      SPECIFIC_HEAT=1.3,
      CONDUCTIVITY=0.2,
      DENSITY=1345.0,
      EMISSIVITY=0.95,
      HEAT_OF_COMBUSTION=2.27E+4,
      N_REACTIONS=1,
      HEAT_OF_REACTION=500.0,
      SPEC_ID(1,1)='Polyester',
      NU_SPEC(1,1)=1.0,
      REFERENCE_TEMPERATURE=100.0/
&MATL ID='FOAM',
      FYI='Properties completely fabricated',
      SPECIFIC_HEAT=1.76,
      CONDUCTIVITY=0.19,
      DENSITY=1100.0,
      HEAT_OF_COMBUSTION=2.27E+4,
      N_REACTIONS=1,
      HEAT_OF_REACTION=800.0,
      SPEC_ID(1,1)='POLYURETHANE',
      NU_SPEC(1,1)=1.0,
      REFERENCE_TEMPERATURE=280.0/

&SURF ID='WALL',
      RGB=200,200,200,
      DEFAULT=.TRUE.,
      MATL_ID(1,1)='GYPSUM PLASTER',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.012/
&SURF ID='UPHOLSTERY',
      FYI='Properties completely fabricated',
      COLOR='PURPLE',
      BURN_AWAY=.TRUE.,
      MATL_ID(1,1)='FABRIC',
      MATL_ID(2,1)='FOAM',
      MATL_MASS_FRACTION(1,1)=1.0,
      MATL_MASS_FRACTION(2,1)=1.0,
      THICKNESS(1:2)=2.5E-3,0.231,
      MASS_FLUX=0.036585,0.0229,
      SPEC_ID='Polyester','POLYURETHANE',
      TAU_MF=1.0,1.0/
&SURF ID='ignitor',
      RGB=255,102,102,
      NET_HEAT_FLUX=15.0,
      RAMP_Q='ignitor_RAMP_Q',
      EMISSIVITY=1.0,
      GEOMETRY='CARTESIAN',
      LENGTH=0.25,
      WIDTH=0.25,
      THICKNESS(1)=1.0/
&RAMP ID='ignitor_RAMP_Q', T=0.0, F=0.0/
&RAMP ID='ignitor_RAMP_Q', T=1.0, F=1.0/
&RAMP ID='ignitor_RAMP_Q', T=180.0, F=1.0/
&RAMP ID='ignitor_RAMP_Q', T=181.0, F=0.0/

&INIT ID='ignitor particle', PART_ID='ignitor particle', N_PARTICLES_PER_CELL=1, CELL_CENTERED=.TRUE., XB=2.4,2.65,4.1,4.35,0.65,0.75/ 

&OBST ID='couch bottom', XB=1.5,3.1,3.8,4.6,0.0,0.4, REAC_ID=Reaction2/ 
&OBST ID='Cushion', XB=1.5,3.1,3.8,4.6,0.4,0.6, BULK_DENSITY=40.0, SURF_ID='UPHOLSTERY'/ 
&OBST ID='Left armrest', XB=1.4,1.5,3.8,4.6,0.7,0.8, BULK_DENSITY=40.0, SURF_ID='UPHOLSTERY'/ 
&OBST ID='Right arm rest', XB=3.1,3.2,3.8,4.6,0.7,0.8, BULK_DENSITY=40.0, SURF_ID='UPHOLSTERY'/ 
&OBST ID='Back rest', XB=1.5,3.1,4.4,4.6,0.6,1.3, BULK_DENSITY=40.0, SURF_ID='UPHOLSTERY'/ 

&VENT ID='Vent #1', SURF_ID='OPEN', XB=1.0,4.0,0.0,0.0,0.0,2.0/ 
&VENT ID='firevent', SURF_ID='ignitor', XB=2.4,2.65,4.1,4.6,0.65,0.65, IOR=-3/ 

&BNDF QUANTITY='RADIATIVE HEAT FLUX'/
&BNDF QUANTITY='CONVECTIVE HEAT FLUX'/
&BNDF QUANTITY='NET HEAT FLUX'/
&BNDF QUANTITY='WALL TEMPERATURE'/
&BNDF QUANTITY='BURNING RATE'/

&SLCF QUANTITY='HRRPUV', CELL_CENTERED=.TRUE., ID='X = 2.5 m: Heat Release Rate per Unit Volume', PBX=2.5/

&TAIL /

**Comment by rmcdermo** [comment]
Your second REAC line does not seem correct.  You have CO and CO2 as reactants.  Create a simple, stand-alone case for just that reaction and make sure it works before putting it together in the complex scenario.

**Reply by bjmali3389** [reply]
Thank you for your reply. I corrected the reac line by removing both CO and CO2, but i am still getting the same error message.

**Comment by mcgratta** [comment]
What is the problem here:
```
FORMULA='CC1H1.7O0.74N0.002.'
```

**Comment by mcgratta** [comment]
Even if you correct that, you'll next have to deal with unbalanced stoichiometry.

**Comment by mcgratta** [comment]
This [PR](https://github.com/firemodels/fds/pull/12700) hopefully will trap the error in the FORMULA parsing.

**Comment by bjmali3389** [comment]
[couchex4.txt](https://github.com/firemodels/fds/files/14800029/couchex4.txt)
 Thank you for your kind suggestions. I have hopefully corrected the errors in formula and stoichiometry. But i am still getting the same erros. Also, I am unaware of how to use the suggested FORMULA error parsing thread.
I am attaching the updated file for reference

**Comment by drjfloyd** [comment, marked as answer]
As Randy suggested in the first reponse to you. Just make a simple case with just the reaction. Don't worry about the details of geometry and materials. Work this problem one piece at a time.

You have not corrected your FORMULA:

&SPEC ID='Polyester', FORMULA='C1H1.7O0.74N0.002.', SPECIFIC_HEAT=1.3/

Take this input, and just to help you better see what you have typed, add spaces after each element+number pair.  Do you notice any typographical errors? Is there some combination of element and number where you either do not have an element or do not have a properly formatted real numbers?

**Comment by mcgratta** [comment]
FYI, FDS will now flag that `FORMULA` with an error and stop.

**Comment by bjmali3389** [comment]
Thank you for your suggestions. There were some problems in my stoichiometry balance it seems, and once corrected, it is running.
